### ISSUE TITLE
HomeKit Home tile keeps resetting

### ISSUE THREAD

**Issue by nicoodeimos** [jubepue/homebridge-petkit-pet-feeder#6]
**Describe the bug**
Seems like every time I restart Homebridge, this plugin resets the name I have given to all my services in the Home app.
I always rename and change the default room of all accessories published by the plugin, but it keeps getting reset somehow and gets the default name "Fresh Element Mini".
Seems like the accessory is reset, and removed/added again, because all related automations are removed just like they would when I remove manually another accessory.

**Detail Information**
NodeJS version: v16.17.1
Homebridge version: v1.5.0
Plugin version: v3.2.1

** Screenshot **
![image](https://user-images.githubusercontent.com/3419988/193810473-6bb2e5ef-ab38-4b33-89b3-e8e970191dc3.png)

**Comment by jubepue** [comment]
Yes, with the new update of iOS homekit i have seen this problem, i have working hard to correct some problems

jubepue

**Comment by nicoodeimos** [comment]
Have you found a way to bypass this?

**Comment by jubepue** [comment]
Hi,
Have you reinstalled the plugin and paired the bridge again? this has been solved too?

jubepue

**Comment by nicoodeimos** [comment]
I have, I’ll let you know if the name resets in the coming days! 
‘thanks

**Comment by nicoodeimos** [comment]
Sorry, just enabled a setting from the config file (desiccant monitoring) and restarted my homebridge instance: the tile has been reset and I have lost all my names again 😌

**Comment by nicoodeimos** [comment]
Hello @jubepue did you get a chance to have a look? I installed a new plugin yesterday and my tiles reseted again. Many thanks !

**Comment by nicoodeimos** [comment]
Little up @jubepue, can you let me know if this issue can be resolved as this bug is pretty annoying and sometimes prevents pets to be fed because accessory and automations reset.

I’ll gladly help you in this process. Just let me know. Many thanks!

**Comment by yclai1229** [comment]
Me too, is there a solution?

**Comment by jubepue** [comment]
Hello again,
With the new update of homekit, when restarting homebridge does not change the names but it is true that when incorporating a new option they change. I'm going to make changes asap. I'll upload an update to fix it.

Regards
Jubepue

**Comment by nicoodeimos** [comment]
Hello there.
Thank you for this! On my side, it is not just the name, the whole accessory resets and looses its room, automations and name.

**Comment by jubepue** [comment]
I have not abandoned it but for important reasons I have had to prioritize other things. The first week of February I start with it.

jubepue

**Comment by jubepue** [comment]
Hello!
Sorry for the delay but I was working on a new plugin to integrate all the petkit accessories in a single platform, it detects the accessories and creates, deletes and updates them automatically. You can install it on your homebridge by searching for "homebridge-petkit-platform". currently only works with feeder models but soon, eversweet and pura. https://github.com/jubepue/homebridge-petkit-platform
